package.json

```diff
@@ -25,7 +25,7 @@
     "build:iife": "esbuild src/js/index.js --bundle --target=es2019 --format=iife --outdir=dist/iife --minify --sourcemap --global-name=MediaChrome && yarn size",
     "build:react": "node ./scripts/react/build.js",
     "build:manifest": "npx @custom-elements-manifest/analyzer analyze --config scripts/custom-elements-manifest.config.js",
-    "build": "yarn build:esm && yarn build:cjs && yarn build:iife && yarn build:react && yarn build:types && yarn build:manifest",
+    "build": "yarn build:types && yarn build:esm && yarn build:cjs && yarn build:iife && yarn build:react && yarn build:manifest",
```

Comment:
NOTE: switch the order to ensure types are available before build (for dev watch behavior/expectations)